src/mkdocs_print_site_plugin/plugin.py

```diff
@@ -330,6 +330,141 @@ def on_post_build(self, config, **kwargs):
+                            return f"#{anchor}", title
+                        else:
+                            # Try to find a similar anchor (case-insensitive, partial match)
+                            anchor_lower = anchor.lower()
+                            for available_anchor in available_anchors:
+                                if (available_anchor.lower() == anchor_lower or 
+                                    anchor_lower in available_anchor.lower() or 
+                                    available_anchor.lower() in anchor_lower):
+                                    return f"#{available_anchor}", title
+                            
+                            return f"#{anchor}", title  # Return original anchor anyway
+                    else:
+                        # If no anchor in original URL, try fuzzy matching with identifier
+                        identifier_lower = identifier.lower()
+                        for available_anchor in available_anchors:
+                            if (available_anchor.lower() == identifier_lower or 
+                                identifier_lower in available_anchor.lower() or 
+                                available_anchor.lower() in identifier_lower):
+                                return f"#{available_anchor}", title
+                        
+                        return f"#{identifier}", title  # Return anyway, might work
+                        
+                except Exception as e:
+                    # Fallback: check if identifier exists as anchor or find fuzzy match
+                    if identifier in available_anchors:
+                        return f"#{identifier}", identifier
+                    else:
+                        # Try fuzzy matching as fallback
+                        identifier_lower = identifier.lower()
+                        for available_anchor in available_anchors:
+                            if (available_anchor.lower() == identifier_lower or 
+                                identifier_lower in available_anchor.lower()):
+                                return f"#{available_anchor}", identifier
+                        return f"#{identifier}", identifier  # Return anyway as last fallback
+            
+            # Apply cross-references to the HTML
+            html, unmapped = fix_refs(
+                html,
+                print_page_url_mapper,
+                link_titles=autorefs_plugin._link_titles,
+                strip_title_tags=autorefs_plugin._strip_title_tags,
+                _legacy_refs=autorefs_plugin.legacy_refs,
+            )
+            if unmapped:
+                logger.warning(f"[mkdocs-print-site] Unmapped autorefs: {[ref for ref, _ in unmapped]}")
+        else:
+            logger.info("[mkdocs-print-site] No autorefs plugin found")   
+
+        # Alternative autorefs implementation when plugin isn't available
+        if not autorefs_plugin:
```

Comment:
Here we are rebuilding autorefs functionality. Let's remove this entire section, and just use the implementation for when autorefs plugin is activated.

Reply:
Removed.